src/pages/events/createEvents/createEventCriteria/BonusPercentage.tsx

```diff
@@ -0,0 +1,188 @@
+import React, {useContext} from 'react';
+import {Box, Grid, IconButton, InputBase, Paper, Typography} from "@mui/material";
+import {IBonusPercentageParams, IEvaluatee} from "../TypesEvents";
+import EditIconCalculationCard from "../../../../assets/images/Icons/EditIconCalculationCard";
+import DollarIcon from "../../../../assets/images/Icons/DollarIcon";
+import PercentIcon from "../../../../assets/images/Icons/PercentIcon";
+import useCreateEventStyles from "../../../../assets/styleJs/events/createEvent";
+import {EventContext} from "../EventsContext";
+
+const BonusPercentage = (params: IBonusPercentageParams) => {
+
+  const UseEventContext = () => useContext(EventContext)
+  const classes = useCreateEventStyles()
+  const {state} = UseEventContext()
+
+  const changeEditable = (id: string) => {
+    const newArray = params.evaluateesList.map((item: IEvaluatee) => {
+      if (item.id === id) {
+        return {...item, isEditable: true}
+      }
+      return item
+    })
+    params.setEvaluateesList(newArray)
+  }
```

Comment:
you can use  onChangeEvaluateeData to not write double function

Reply:
done